test/parallel/test-crypto-fips.js

```diff
@@ -212,6 +212,15 @@ testHelper(
   'require("crypto").fips = false',
   process.env);
 
+// --force-fips makes setFipsCrypto enable a no-op (FIPS stays on)
+testHelper(
+  compiledWithFips() ? 'stdout' : 'stderr',
+  ['--force-fips'],
+  compiledWithFips() ? FIPS_ENABLED : OPTION_ERROR_STRING,
+  '(require("crypto").fips = true,' +
+  'require("crypto").fips)',
```

Comment:
I personally would use a single line for this argument as all other arguments are also on a single line but that is just taste and does not have to be addressed.

Reply:
I copied this from a test further up. There are a few things I'd change about this test file, but I thought I'd try to keep this diff clean.